Tella/Scenes/Settings/Views/Servers/ServerSelectionView.swift

```diff
@@ -95,14 +94,14 @@ struct ServerSelectionView: View {
     }
     
     fileprivate func navigateToGDriveFlow() {
-        serversViewModel.shouldHideNextButton = true
+        serversViewModel.shouldEnableNextButton = false
```

Comment:
could we add here:

`serversViewModel.selectedServerType = nil`

If the user cancel the sign in flow the drive connection would be kept as selected and the next button disabled

Reply:
<img width="344" alt="Screenshot 2024-08-28 at 2 24 14 PM" src="https://github.com/user-attachments/assets/6e4929d6-5fa4-4c7c-a130-e576c0c15cd2">

Reply:
yeah you're right, I have the same issue
What do you think if we add  `serversViewModel.shouldEnableNextButton = true` to 
```
.onReceive(gDriveVM.$signInState){ signInState in
            if case .error(let message) = signInState {
                serversViewModel.shouldEnableNextButton = true
                Toast.displayToast(message: message)
            }
        }
        

```

Reply:
I'll try also to add `serversViewModel.selectedServerType = nil`

Reply:
I fixed the issue by adding a `resetView`  function  
```
fileprivate func resetView() {
        serversViewModel.selectedServerType = nil
        serversViewModel.shouldEnableNextButton = false
    }

```